aptos-move/framework/move-stdlib/sources/bcs.move

```diff
@@ -13,5 +13,14 @@ module std::bcs {
     spec module {
         /// Native function which is defined in the prover's prelude.
         native fun serialize<MoveValue>(v: &MoveValue): vector<u8>;
+
+        fun spec_to_bytes<MoveValue>(v: MoveValue): vector<u8>;
+    }
+
+
+    spec to_bytes<MoveValue>(v: &MoveValue): vector<u8> {
```

Comment:
If you use `serialize`, you don't need to have this abstract spec. This is already done natively in the boogie level.